android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsViewModel.kt

```diff
@@ -58,4 +61,16 @@ class SurveyQuestionsViewModel(
             "${questionAndAnswers.questionIndex} - ${questionAndAnswers.answerInputs}"
         )
     }
+
+    fun onOpenExitDialogClicked() {
+        _showExitDialog.value = true
+    }
+
+    fun onExitDialogConfirm() {
+        _showExitDialog.value = false
+    }
+
+    fun onExitDialogDismiss() {
+        _showExitDialog.value = false
+    }
```

Comment:
Should we manage `showExitDialog` state in the Composable screen instead?
If we would like to keep the dialog action handling in VM, please add unit test for them 🙏

Reply:
Yep, we can manage that state in the composable screen. Fixed in https://github.com/suho/kmm-ic/pull/122/commits/34df858b7cdc74f4a9252b232270a14b2ef99b47